core-api/src/test/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollectorTest.java

```diff
@@ -542,6 +610,67 @@ void getNonTLSServicePorts() {
         );
     }
 
+    @Test
+    void getNonTLSServicePorts() {
+        underTest.init();
+        assertThat(underTest.getAllServicePorts(CDH_7_2_16, false)).containsOnly(
+                Map.entry("ATLAS", 21000),
+                Map.entry("ATLAS_API", 21000),
+                Map.entry("AVATICA", 8765),
+                Map.entry("CM-API", 7180),
+                Map.entry("CM-UI", 7180),
+                Map.entry("CRUISE-CONTROL", 8899),
+                Map.entry("DAS", 30800),
+                Map.entry("FLINK", 18211),
+                Map.entry("HBASEUI", 16010),
+                Map.entry("HBASEJARS", 16010),
+                Map.entry("HDFSUI", 9870),
+                Map.entry("HIVE", 10001),
+                Map.entry("HUE", 8889),
+                Map.entry("IMPALA", 28000),
+                Map.entry("IMPALA_DEBUG_UI", 25000),
+                Map.entry("JOBHISTORYUI", 19888),
+                Map.entry("JOBTRACKER", 8032),
+                Map.entry("RESOURCEMANAGERAPI", 8032),
+                Map.entry("KAFKA_CONNECT", 28083),
+                Map.entry("KUDUUI", 8051),
+                Map.entry("LIVYSERVER1", 8998),
+                Map.entry("LIVYSERVER_API", 8998),
+                Map.entry("LIVY_FOR_SPARK3", 28998),
+                Map.entry("LIVY_FOR_SPARK3_API", 28998),
+                Map.entry("NAMENODE", 8020),
+                Map.entry("NIFI", 8080),
+                Map.entry("NIFI-REGISTRY", 18080),
+                Map.entry("NIFI_REST", 8080),
+                Map.entry("NIFI-REGISTRY-REST", 18080),
+                Map.entry("OOZIE", 11000),
+                Map.entry("OPDB-AGENT", 8181),
+                Map.entry("DATA-DISCOVERY-SERVICE-API", 21600),
+                Map.entry("PROFILER-ADMIN-API", 21700),
+                Map.entry("PROFILER-METRICS-API", 21800),
+                Map.entry("PROFILER-SCHEDULER-API", 21900),
+                Map.entry("RANGER", 6080),
+                Map.entry("SCHEMA-REGISTRY", 7788),
+                Map.entry("SCHEMA-REGISTRY-API", 7788),
+                Map.entry("SMM-API", 8585),
+                Map.entry("SMM-UI", 9991),
+                Map.entry("SOLR", 8993),
+                Map.entry("SPARKHISTORYUI", 18088),
+                Map.entry("SPARK3HISTORYUI", 18089),
+                Map.entry("SSB-SSE-UI", 18121),
+                Map.entry("SSB-SSE-WS", 18121),
+                Map.entry("SSB-SSE-UI-LB", 18121),
+                Map.entry("SSB-SSE-WS-LB", 18121),
```

Comment:
You had set these to `8445` in [exposed-services.json](https://github.com/hortonworks/cloudbreak/pull/12864/files#diff-7ca3fcf95baaf86235de4bb37c5c44707036ce30d8ec2099c64ea31c18a1fe73)